keyboards/e6_rgb/config.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * config.h
+ *
+ */
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xDEAD
+#define PRODUCT_ID      0xFEED
+#define DEVICE_VER      0x0062
+#define MANUFACTURER    astro
+#define PRODUCT         e6rgb
+#define DESCRIPTION     60% rgb keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
+/* key matrix pins */
+#define MATRIX_ROW_PINS { F1, F4, F5, F6, D6 }
+#define MATRIX_COL_PINS { D7, B4, B5, B6, C6, C7, F7, F0, B0, B1, D2, D3, B3, B2 }
+#define UNUSED_PINS
+//#define DIODE_DIRECTION ROW2COL
+#define DIODE_DIRECTION COL2ROW
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* key combination for magic key command */
+#define IS_COMMAND() ( \
```

Comment:
This is no longer required. Please remove lines 32–36 (inclusive) in this file.

Reply:
thanks, it was removed